examples/complete/fixtures.us-east-2.tfvars

```diff
@@ -6,7 +6,80 @@ environment = "ue2"
 
 stage = "test"
 
-name = "example"
+name = "amplify"
 
-example_input = "Hello, world!"
+platform = "WEB"
 
+repository = "https://github.com/cloudposse/amplify-test2"
+
+iam_service_role_enabled = false
+
+enable_auto_branch_creation = false
+
+enable_branch_auto_build = true
+
+enable_branch_auto_deletion = true
+
+enable_basic_auth = false
+
+access_token = "test"
```

Comment:
Let's store the access token in SSM and fetch it in `main.tf` if that is all that is keeping Terratest from working.